code/go/0chain.net/smartcontract/dbs/event/eventdb.go

```diff
@@ -15,17 +20,81 @@ func NewEventDb(config dbs.DbAccess) (*EventDb, error) {
 		return nil, err
 	}
 	eventDb := &EventDb{
-		Store: db,
+		Store:          db,
+		eBufferChannel: make(chan eventCtx, 100),
+		eChannel:       make(chan eventCtx, 100),
```

Comment:
what is an idea behind this second channel?

Reply:
This was an experiment I and @Sriep were trying ... changing that.. 😓